symplyphysics/laws/waves/phase_difference_of_oscillations_of_two_points.py

```diff
@@ -0,0 +1,48 @@
+from sympy import (Eq, solve, pi, S)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    convert_to,
+    dimensionless,
+)
+
+# Description
+## The phase difference of the oscillations of two points spaced at distances r1 and r2 from the source of
+## the oscillations depends on these distances and wavelength.
+
+## Law is: phi = 2 * pi * (r2 - r1) / L, where
```

Comment:
It seems, from choosing which is the first point, depends the sign of the result. Is there any way to choose first point, so the sign does not change arbitrarily? Or maybe it is better to use absolute value of difference of distances?

Reply:
> It seems, from choosing which is the first point, depends the sign of the result. Is there any way to choose first point, so the sign does not change arbitrarily? Or maybe it is better to use absolute value of difference of distances?

Changed the code - added Abs().
from sympy import Abs
law = Eq(phase_difference, 2 * pi * Abs(distance_second_point - distance_first_point) / wavelength)

Now I get the following error.
![image](https://github.com/blackyblack/symplyphysics/assets/100017508/39f3d9e8-de8e-4a74-b8b5-a8eec49ab781)

Reply:
Will look into it.